parsers/CA_NS.py

```diff
@@ -37,21 +37,37 @@ def fetch_production(country_code='CA-NS', session=None):
     }
     """
     r = session or requests.session()
-    url = 'http://www.nspower.ca/system_report/today/currentmix.json'
-    response = r.get(url)
+    mix_url = 'http://www.nspower.ca/system_report/today/currentmix.json'
+    mix_data = r.get(mix_url).json()
+
+    load_url = 'http://www.nspower.ca/system_report/today/currentload.json'
+    load_data = r.get(load_url).json()
+
     data = []
-    for obj in response.json():
+    for mix in mix_data:
+        corresponding_load = [load_period for load_period in load_data
+                              if load_period['datetime'] == mix['datetime']]
+
+        # in mix_data, the values are expressed as percentages,
+        # e.g. "Solid Fuel":  45.76 meaning 45.76%
+        # Divide load by 100.0 to simplify calculations later on.
```

Comment:
Well spotted! Do we have data for generation rather than load? Maybe by knowing about exchanges?